src/styles/card-block.scss

```diff
@@ -0,0 +1,108 @@
+.main {
```

Comment:
even if it's a small block, according to BEM rules each block should be in a separate file